src/hooks/spaces/useSpaceById.ts

```diff
@@ -7,16 +7,16 @@ import { SpaceId, SpaceWithId, WorldId } from "types/id";
 
 import { useFireDocument } from "hooks/fire/useFireDocument";
 
-type UseSpaceById = (
-  options: string | undefined
-) => LoadStatus & {
+type UseSpaceById = (options: {
+  spaceId: string | undefined;
+}) => LoadStatus & {
   space?: SpaceWithId;
   spaceId?: SpaceId;
   worldId?: WorldId;
   error?: string;
```

Comment:
[OOS]
[NOTE] to self: 

`LoadStatus` now has `error` field as well. It should probably be that one relayed from `useFireDocument` to callers of `useSpaceById`